### ISSUE TITLE
Looping for items in crafting slots

### ISSUE THREAD

**Issue by Eryk1983S** [SkriptLang/Skript#1943]
**Engine:** Spigot 1.12.2.-R.0.1-SNAPSHOT
**Skript-Version:** 2.2. dev37c

Which expression I should use to loop items in the crafting slots? My purpose is to set different events or conditions or different crafting recipes. For example: The crafting of a bed from planks and wool shall depend of condition X and crafting of a bed from a bed and a dye from condition Y.

Thanks in advance for any help

**Comment by Blueyescat** [comment]
First of all you really should use the latest version of Skript for 1.9.4+.

---
I'm not sure about what you mean but you can get the ingredient slots using the slot expression, for example: `slot 1 of player's current inventory`. Ingredient slots are 1-9 and the result slot is 0. You can use a slot like an item e.g. `if <slot> is a wool:`

**Comment by Eryk1983S** [comment]
First of all - as I updated the version, I got a lot of errors because of the different item names in 1.12. and 1.13. That was with Skript 2.3.6. and 2.3.2. Really, I don't want to rewrite all scripts only to have the newest version - probably for a while :)

I'll try your hint out and share the results later (please don't close the thread yet). As I remember from yesterday trials, similiar expressions didn't work, but especially in Skript is "similiar" not "the same". I used "player's inventory" and it looped all, but not the crafting slots (0-9). Maybe this word "current" will be the key. I'll let you all know...

**Comment by Blueyescat** [comment]
Yeah `current inventory` will be the target inventory.

It is not really hard to fix the items names and i don't think that there are many differences. **If** you don't listen to warnings and use item IDs, sure.

**Comment by Eryk1983S** [comment]
I don't use the ID's, but anyway I got some errors. I'll take a closer look and maybe I'll describe it in another thread.

And to the main topic - I used the following trial-script:
on craft:
	event-item is bed
	**loop items in slots 1-9 of player's current inventory:**
		loop-item is bed:
			send "Message 1" to the player
		loop-item is wool:
			send "Message 2" to the player

I made a bed from another bed and from a dye --> I got Message 1 --> correct
But as I made a bed from planks and wool, I still got Message 1, not 2.

It seems, the script always recieved a bed (event-item or loop-item from slot 0). Is there something wrong with the bolded line?
